Views/PortalMenuWidget.cshtml

```diff
@@ -0,0 +1,84 @@
+@using Lombiq.HelpfulExtensions.Extensions.Widgets
+@using OrchardCore.Navigation
+
+<style asp-name="font-awesome"></style>
+
+@{
+    var useWrapper = !(Model.NoWrapper as bool? ?? false);
+    IEnumerable<MenuItem> menuItems = Model.MenuItems;
+
+    const string blockName = WidgetTypes.PortalMenuWidget;

```

Comment:
The convention for constants here is PascalCased or at least this is how we did it in other cshtml files.

Reply:
The PascalCase naming of local constants has always bothered me, because it's not consistent with other local variables. I'm in favor of using CamelCase like here, but I guess that would be another bike-shed we might want to avoid right now.

Reply:
It's a convention you have to actively remember to go against other more generic conventions and I forgot, so I'm not a fan of it. I say let's go behind that bike-shed Oliver mentioned and duke it out ;)
On a more serious note, do we have this documented somewhere? If not, where should we?

Reply:
Also if I turn the constant PascalCase, then it's weird with the similarly named camelCase variable next to it, for example:
`<a id="@dropdownIdName" class=" @DropdownElementName"` (sample pruned to better drive the point home)

Reply:
Now, that we have a project-wide agreement, we can bring it to the General Developers table 😀

Reply:
Agreed.